### ISSUE TITLE
Space return, throws, case rule flagging "default"

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#442]
The rule is flagged `default` in a `switch` statement as needing to have a space after it. This triggers the warning:

``` js
switch(foo) {

    default: return;
}
```
